### PR TITLE
Update readme to include migration guide for users on the 0.x helm chart

### PR DESCRIPTION
**Pull request by bruce-y**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +40,68 @@ helm repo update
...
-  --set superblocks.agentEnvironment='<"*"|"staging"|"production">' \
+  --set superblocks.agentHostUrl='http[s]://<agent-host[:port]>' \
+  --set superblocks.agentTags='<"profile:*"|"profile:staging"|"profile:production">' \
   --set superblocks.agentDataDomain='<"app.superblocks.com"|"eu.superblocks.com">'
 ```
 
+#### v0.x to v1.0 Migration Guide
+
+* Container ports are now specified under `service.ports.http`. It is recommended that you leave these settings as default.
+* Agent URL field `superblocks.agentUrl` must be updated from `https://myagent.example.com/agent` to `https://myagent.example.com`. The new OPA no longer has the `agent` prefix to its endpoints.
```
Comment (gzysk8) [inline README.md:51]:
```suggestion
* Agent URL field `superblocks.agentUrl` must be updated from `https://myagent.example.com/agent` to `https://myagent.example.com`. The new OPA no longer has the `agent` suffix path on its endpoints.
```

File: README.md [hunk 2]
```diff
@@ -41,11 +40,68 @@ helm repo update
...
+* Agent Environment variable `superblocks.agentEnvironment` is deprecated and you must start using `superblocks.agentTags`.
+  * `superblocks.agentEnvironment: "*"` is equivalent to `superblocks.agentTags: "profile:*"`
+  * `superblocks.agentEnvironment: "staging"` is equivalent to `superblocks.agentTags: "profile:staging"`
+  * `superblocks.agentEnvironment: "production"` is equivalent to `superblocks.agentTags: "profile:production"`
+* The `controller` block in the values file is deprecated. Everything underneath can be specified at one level higher. You can simply delete the `controller` block and unindent everything underneath it.
```
Comment (gzysk8) [inline README.md:56]:
```suggestion
* The `controller` block in the values file is deprecated. Everything underneath can be specified one level higher. You can simply delete the `controller` block and unindent everything underneath it.
```

File: README.md [hunk 3]
```diff
@@ -55,15 +111,15 @@ To do so, first export the agent configuration.
 ```sh
 # obtained during agent onboarding
 export SUPERBLOCKS_AGENT_KEY='<agent-key>'
-export SUPERBLOCKS_AGENT_HOST_URL='http[s]://<agent-host[:port]>/agent'
-export SUPERBLOCKS_AGENT_ENVIRONMENT='<"*"|"staging"|"production">'
+export SUPERBLOCKS_AGENT_HOST_URL='http[s]://<agent-host[:port]>'
```
Comment (gzysk8) [inline README.md:114]: Do we still need to call out the port? Or does the syntax mean it's optional?

Comment (bruce-y) [reply README.md:114]: Yeah, the syntax makes it optional. It's only needed if it's not on the default http/https ports

**Review by gzysk8 — APPROVED** [review]

**Pull request merged**